static/js/tools/map/stat_var_chooser.tsx

```diff
@@ -114,10 +114,11 @@ export function StatVarChooser(props: StatVarChooserProps): JSX.Element {
       selectSV={(svDcid) =>
         selectStatVar(dateCtx, statVar, display, placeInfo, svDcid)
       }
-      numEntitiesExistence={Math.min(
-        NUM_ENTITIES_EXISTENCE,
-        samplePlaces.length
-      )}
+      numEntitiesExistence={
```

Comment:
Can you extract this logic to a function and add some comments explaining the isCustomDC vs NUM_ENTITIES_EXISTENCE check

Reply:
Good idea. Extracted to a helper function and added an explaining comment.